README.adoc

```diff
@@ -39,7 +39,16 @@ if it can find this file in the current directory. If it can't find this file in
 look in `..` too.
 This is a useful feature if you want to de-clutter your command line.
 
-Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties.
+Chronicle Core class `Jvm` loads the file `system.properties` into the System's properties. To ensure it is loaded early enough, add the following code to your Main
```

Comment:
"add the following code to your Main" -> "you can load in a static block"

Also please mention how Jvm.getInteger etc work and how they ensure that system.properties are loaded